tests/test_inference.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+import model_inference.inference as inference
+
+import asyncio
+
+box1 = {
+    "topX": 1,
+    "topY": 1,
+    "bottomX": 40,
+    "bottomY": 40,
+}
+box2 = {
+    "topX": 20,
+    "topY": 20,
+    "bottomX":60,
+    "bottomY": 40,
+}
```

Comment:
I would suggest using the `setUp` function from unittest.TestCase to set up the boxes.

Reply:
@MaxenceGui please take back this pull request and fix it to merge it in.